.github/workflows/release-egopulse.yml

```
@@ -0,0 +1,168 @@
+        with:
+          fetch-depth: 0
+
+      - name: Download artifacts
+        uses: actions/download-artifact@v4
+        with:
+          path: dist
+          merge-multiple: true
+
+      - name: Generate checksums
+        shell: bash
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2035:info:2:11: Use ./*glob* or -- *glob* so names with dashes won't become options

(shellcheck)